cloudos_cli/__main__.py

```diff
@@ -208,6 +208,58 @@ def get_param_value(ctx, param_value, param_name, default_value, required=False,
     return result
 
 
+# This is put here to avoid changing current code.
+# Once we have all three (Bitbucket, Github, Gitlab) repo systems
+# working with API keys, we can re-arrange the UI
+@run_cloudos_cli.group()
+def import_workflows():
+    """Import workflows"""
+    print(import_workflows.__doc__ + "\n")
+
+
+@import_workflows.command("gitlab")
+@click.option('-k',
+              '--apikey',
+              help='Your CloudOS API key',
+              required=True)
+@click.option('-c',
+              '--cloudos-url',
+              help=('The CloudOS url you are trying to access to. ' +
+                    'Default=https://cloudos.lifebit.ai.'),
+              default='https://cloudos.lifebit.ai')
+@click.option('--workspace-id',
+              help='The specific CloudOS workspace id.',
+              required=True)
+# This is a placeholder in case we decide to handle all imports from the same function
+# @click.option("-p", "--platform", type=click.Choice(["github", "gitlab", "bitbucket"]),
+#               help="Repository service where the workflow is located. Valid choices: github, gitlab, bitbucket",
+#               default="github")
+@click.option("-r", "--repository-apikey", help="Repository API key.", required=True)
+@click.option("-n", "--workflow-name", help="Workflow name.", required=True)
+@click.option("-w", "--workflow-url", help="URL of the workflow repository.", required=True)
+@click.option("-d", "--workflow-docs-link", help="URL to the documentation of the workflow.", default='')
+@click.option('--disable-ssl-verification',
+              help=('Disable SSL certificate verification. Please, remember that this option is ' +
+                    'not generally recommended for security reasons.'),
+              is_flag=True)
+@click.option('--ssl-cert',
+              help='Path to your SSL certificate file.')
+def import_gitlab(apikey, cloudos_url, workspace_id, workflow_name, workflow_url, workflow_docs_link,
```

Comment:
```suggestion
@click.option('--profile', help='Profile to use from the config file', default=None)
@click.pass_context
def import_gitlab(ctx, apikey, cloudos_url, workspace_id, workflow_name, workflow_url, workflow_docs_link,
```
It would be great if you can also add the `--profile` parameter so we can use URL, workspace-id and apikey from the config files from the user